tests/strings/repeat.du

```diff
@@ -0,0 +1,17 @@
+/**
```

Comment:
We will need to make sure we import this new file to the tests/strings/import.du file, there's currently no smart file resolution for the tests

https://github.com/dictu-lang/Dictu/blob/develop/tests/strings/import.du